src/interfaces/IDelegation.sol

```diff
@@ -1,6 +1,18 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity >=0.5.0;
 
+struct Delegation {
+    address delegatee;
+    uint256 nonce;
```

Comment:
Not sure about it, as the struct is already called delegation
```suggestion
    uint256 delegationNonce;
```

Reply:
Nonce is fine
Because `delegation.nonce` is explicit